modules/lvm2/udiskslinuxvdovolume.c

```diff
@@ -430,6 +430,90 @@ handle_resize_physical (UDisksVDOVolume       *_volume,
   return TRUE;
 }
 
+static void
+stats_add_element (const gchar *key, const gchar *value, GVariantBuilder *builder)
+{
+  g_variant_builder_add (builder, "{ss}", key, value);
+}
+
+static gboolean
+handle_get_statistics (UDisksVDOVolume       *_volume,
+                       GDBusMethodInvocation *invocation,
+                       GVariant              *options)
+{
+  const gchar *pool_path = NULL;
+  UDisksObject *pool_object = NULL;
+  UDisksLinuxLogicalVolumeObject *object = NULL;
+  UDisksLinuxVolumeGroupObject *group_object = NULL;
+  GError *error = NULL;
+  UDisksDaemon *daemon = NULL;
+  GHashTable *stats = NULL;
+  GVariantBuilder builder;
+  uid_t caller_uid;
+  const gchar *pool_name = NULL;
+  const gchar *vg_name = NULL;
+
+  object = udisks_daemon_util_dup_object (_volume, &error);
+  if (object == NULL)
+    {
+      g_dbus_method_invocation_take_error (invocation, error);
+      goto out;
+    }
+
+  if (!common_setup (object, invocation, options,
+                     N_("Authentication is required to resize a VDO volume"),
```

Comment:
copy pasta -- **resize**

Reply:
Fixed. I've actually removed the code, we don't need to check for permissions for a function that only reads some data from `/sys`.